packages/create-demand-bloc-module/src/generators/app/index.js

```diff
@@ -0,0 +1,50 @@
+const path = require('path')
+const packageJson = require('../../../../create-demand-bloc-module/package.json')
+const Generator = require('yeoman-generator')
+
+const latestDeps = ['@karhoo/demand-api']
+
```

Comment:
I think it will be good idea to provide also optional deps (select in promting) like demand-deeplink while generate new template. How do you think?

Reply:
That's nice idea for the future